tests/pyrog/migrations/conftest.py

```diff
@@ -0,0 +1,8 @@
+import pytest
+
+
+@pytest.fixture
+def state(request, migrator):
+    marker = request.node.get_closest_marker("migration")
+    target = (marker.kwargs["app_label"], marker.kwargs["migration_name"])
+    return migrator.apply_initial_migration(target)
```

Comment:
does this apply all migrations up to `target` (not included) ?

Reply:
Yes this applies all migration up to `target` included